getting-started/getting-started.md

```diff
@@ -245,43 +300,58 @@ After applying these steps, the relevant portions of your build file will look l
 <build>
 ```
 {% endcode %}
-{% endtab %}
 
-{% tab title="Gradle" %}
+To double-check that everything is working, run the command `./mvnw rewrite:run`. Your project should be upgraded to Spring Boot 2 and all of the test classes should be updated to JUnit 5. Your `pom.xml` file will also have had its Spring dependencies updated, the JUnit 4 dependency removed, and the JUnit 5 dependency added.
+
+### Gradle + External Modules
+
+Unlike Maven projects, Gradle projects have two options for specifying versions for recipes. You can:
+
+1. Add `rewrite-recipe-bom` as a dependency
```

Comment:
```suggestion
1. Add `rewrite-recipe-bom` as a [bill of materials (BOM) dependency](https://docs.gradle.org/current/userguide/platforms.html#sub:bom_import)
```

Adding rewrite-recipe-bom as a regular dependency is not the same as using it to manage the versions of other dependencies. You accurately explain the difference below, but I don't want a discrepancy to cause confusion.

Reply:
Good call out and appreciate the suggested change.